core/src/main/java/com/homeaway/streamplatform/streamregistry/exceptions/BaseStreamException.java

```diff
@@ -27,7 +27,8 @@ public BaseStreamException(Throwable cause, String streamName) {
         this.streamName = streamName;
     }
 
-    public BaseStreamException(String streamName) {
+    public BaseStreamException(String streamName, String message) {
```

Comment:
earlier it was just accepting streamName which was calling super(message).

When the exception was handled, the e.getMessage was null.